js-miniapp-sample/src/routes.js

```diff
@@ -62,6 +64,12 @@ const appItems = [
     navLink: '/fetch_id',
     element: <UuidFetcher />,
   },
+  {
+    icon: <ShoppingCartIcon />,
+    label: 'InApp Purchases',
+    navLink: '/inapp_purchases',
+    element: <Purchases />,
```

Comment:
Is it possible to check & display this icon IF the platform is Android ? Because we won't be supporting this feature in iOS for sometime

Reply:
Can we take up this improvement as a part of https://jira.rakuten-it.com/jira/browse/MINI-1737,
I had made the changes to sample app just to verify my js-sdk changes.